wirecard/oxid-ee
# WDPD-27: Update admin tabs

### This PR

* Makes custom admin tabs display a message if there is no data to display
* Bumps module version to 1.0.0

### How to Test

* Login to the admin
* Go to _Extensions → Modules_ and activate the module
* Go to _Shop Settings → Payment Methods_ and activate at least one Wirecard payment method
* Make a purchase using one of the enabled payment methods
* Go to Administer _Orders → Orders_
* Click on the order that was just created and open the _Transaction_ and _Transaction Details_ tabs – they should display content as usual
* Click on the first order (part of the demo data) and open the _Transaction_ and _Transaction Details_ tabs – a text should be displayed saying that this order doesn't have any transactions
